File: security/src/main/java/com/undancer/cloud/SecurityApplication.java
package com.undancer.cloud;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.oauth2.client.EnableOAuth2Sso;

@EnableOAuth2Sso
@SpringBootApplication
public class SecurityApplication {
    public static void main(String[] args) {
        SpringApplication.run(SecurityApplication.class, args);
    }
}

File: security/src/main/kotlin/com/undancer/cloud/SecurityApplication.kt
package com.undancer.cloud

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@EnableDiscoveryClient
class SecurityApplication : SpringBootServletInitializer()

fun main(args: Array<String>) {
    runApplication<SecurityApplication>(*args)
}